_posts/2020-11-09-ten-tips-for-tidy-data.markdown

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title:  "Ten Tips for Tidy Data"
+date:   2020-11-09 12:54:04 -0700
+categories: tidy data
+---
+
+# Why use tidy data?
+
+So, you’re beginning to explore the wonderful world of bioinformatics! There is an incredible amount of computations and analytics you can accomplish with even just basic bioinformatics tools. However, all of these tools rely on a reliable organization of data spreadsheets in order to do what they’re intended to do and avoid major errors. This is what is known as Tidy Data! Ensuring the use of tidy data not only prevents errors in data analysis, it also facilitates effective communication among collaborators. Below, I’ve laid out ten basic principles of tidy data. By using these principles, you’re setting up a strong foundation to make solid, clear, and reproducible analyses in the future.
```

Comment:
Not sure how official the post should be. If it were to be official and formal, I wouldn't start the first paragraph with 'so'.

Reply:
Right, I'm not sure how formal to sound either. It's easy enough to be cautious and remove "so"